themes/custom/lex/scss/molecules/_contact_card.scss

```diff
@@ -0,0 +1,97 @@
+.contact-card {
+    width: 265px;
+    background-color: #EFEFEF;
+    text-align: center;
+    display: inline-block;
+
+
+    .contact-img-container {
+        width: 200px;
+        height: 200px;
+        border-radius: 50%;
+        margin: 20px auto 10px;
+        overflow: hidden;
+        display: inline-block;
+    }
+
+    .contact-img {
+        max-width: none;
+        left: -40px;
+        position: relative;
+        display: inline-block;
+    }
+
+    .contact-heading {
+        color: #0558A7;
+        font-size: 18pt;
+        font-family: 'Open Sans';
```

Comment:
Should be able to use the variables in fonts.scss

Reply:
We have global fonts stuff too. So you should mostly be able to wrap it in whatever h tag you need to get the intended styles instead of redoing everything that's done already.